variables.tf

```diff
@@ -1,3 +1,15 @@
+variable "LOG_LEVEL" {
+  description = "Log level"
+  default = "DEBUG"
```

Comment:
I personally tend to suggest defaulting to INFO - DEBUG typically holds more info than I want turned on all the time - but I know different people/teams would disagree with that!